debug.go

```diff
@@ -36,7 +36,7 @@ type (
 	}
 
 	ExportableLeafNode struct {
-		Stem   []byte   `json:"stem"`
+		Stem   ExtStem  `json:"stem"`
```

Comment:
not sure if that works out of the box (it might, haven't tested it). Just to make sure, please add a test that deserializes this structure in `debug_test.go`.